coalib/core/DependencyTracker.py

```diff
@@ -293,7 +293,7 @@ def check_circular_dependencies(self):
          ...
         coalib.core.CircularDependencyError.CircularDependencyError: ...
 
-        :raises coalib.core.CircularDependencyError.CircularDependencyError:
+        :raises CircularDependencyError:
```

Comment:
@Makman2 Do we use sphinx autodoc for generating API docs? Then I would first take a look if the link to the exception class doc is correctly generated if we use the short form here...

cc @thisch